### PR TITLE
switch to env provided seed nodes

### PR DESCRIPTION
**Pull request by ktoso**
Progress towards running the sharding bench in the cloud

### CODE REVIEW

File: sharding/src/main/resources/application.conf [hunk 1]
```diff
@@ -3,11 +3,17 @@ akka {
   extensions = ["com.romix.akka.serialization.kryo.KryoSerializationExtension$"]
 
   cluster {
+    
+    # !! IMPORTANT !!
+    # the below are provided by root-application.conf on our cloud nodes, 
+    # these values here serve only as local fallback
     seed-nodes = [
       "akka.tcp://cluster@127.0.0.1:2551",
```
Comment (patriknw) [inline sharding/src/main/resources/application.conf]: we should use Artery, i.e. "akka://"

File: sharding/src/main/scala/com/lightbend/akka/bench/sharding/ShardingLatencyApp.scala [hunk 2]
```diff
@@ -3,24 +3,30 @@
  */
 package com.lightbend.akka.bench.sharding
 
-import akka.actor.{ActorSystem, Props}
+import java.io.File
+
+import akka.actor.{ ActorSystem, Props }
 import com.typesafe.config.ConfigFactory
 
 object ShardingLatencyApp extends App {
```
Comment (patriknw) [inline sharding/src/main/scala/com/lightbend/akka/bench/sharding/ShardingLatencyApp.scala]: btw, should we include cluster management to dogfood and might be useful ?

File: sharding/src/main/resources/application.conf (hunk 1 above)
Comment (ktoso) [inline sharding/src/main/resources/application.conf]: right, will do so here too for clarity. (Note though that these are overriden by the root-application.conf)

File: sharding/src/main/scala/com/lightbend/akka/bench/sharding/ShardingLatencyApp.scala (hunk 2 above)
Comment (ktoso) [inline sharding/src/main/scala/com/lightbend/akka/bench/sharding/ShardingLatencyApp.scala]: Yes, awesome idea :)

File: infra/nodes/akka-node-015.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    
+    "java": {
+      "jdk_version": 8,
+      "oracle": {
+        "accept_oracle_download_terms": true
+      },
+      "url": "http://download.oracle.com/otn-pub/java/jdk/8u112-b15/server-jre-8u112-linux-x64.tar.gz"
+    },
+    "run_list": [
+        "role[benchmark]",
```
Comment (2m) [inline infra/nodes/akka-node-015.json]: Nodes 010-015 should have an additional `benchmark-ddata` role

Comment (ktoso) [inline infra/nodes/akka-node-015.json]: we can add EXTRA_RUNLIST i guess then

**Pull request merged**